experimental/handoff-library/csharp_dotnetcore/Microsoft.Bot.Builder.HandoffBot/HandoffBot.sln

```diff
@@ -0,0 +1,25 @@
+
+Microsoft Visual Studio Solution File, Format Version 12.00
+# Visual Studio 15
```

Comment:
I would generall expect to find the .sln file and the .csproj files at different places in the tree. Having them as peers is different than all of our other bits.